herbridge/frontend/src/lib/api.js

```diff
@@ -31,8 +31,24 @@ const getResources = (polygon) => {
   )
 }
 
+const submitSingle = (submission) => {
+  return (
+    axios.post('/api/eamena/submit', submission)
+      .then(response => response.data)
+      .catch(error => {
+        throw httpErrorFromAPIError(error)
+      })
+  )
+}
+
+const submit = (submissions) => {
+  const promises = submissions.map(submission => submitSingle(submission))
```

Comment:
Does JS let you do this pointfree style? Like `const promises = submissions.map(submitSingle)`?